.eslintrc

```diff
@@ -7,6 +7,7 @@
     "plugin:prettier/recommended"
   ],
   "rules": {
-    "@typescript-eslint/no-unused-vars": ["warn", { "argsIgnorePattern": "^_" }]
+    "@typescript-eslint/no-unused-vars": ["warn", { "argsIgnorePattern": "^_" }],
+    "@typescript-eslint/no-var-requires": 0
```

Comment:
we don't need this once we remove keyv right?

Reply:
yeah we won't need no-var-requires anymore, but the other rule was already there